flink-core/src/test/java/org/apache/flink/util/concurrent/FixedRetryStrategyTest.java

```diff
@@ -49,6 +49,5 @@ public void testRetryFailure() {
                                 new FixedRetryStrategy(0, Duration.ofMillis(5L))
                                         .getNextRetryStrategy())
                 .isInstanceOf(IllegalStateException.class);
-        ;
```

Comment:
This change should belong to the previous hotfix commit.

Reply:
Good catch!